device/iot-device-samples/device-reconnection-sample/readme.md

```diff
@@ -1,56 +1,63 @@
 # Device Reconnection Sample
 
-This sample code demonstrates the various connection status changes and connection status change reasons the Device Client can return, and how to handle them.
-
-The device client exhibits the following connection status changes with reason:
-
-<table>
```

Comment:
I am wondering why this table gets removed. I think listed connection status & explanation is still useful (especially to new users) as a reference even after this reworking.

Reply:
I feel that showing is better than telling here. The connection status callback that we define in this sample synthesizes this table into the actual code that a user should write. The contents of this table may make for good javadocs on the connection status change reason enums themselves, though